ruby/lib/svix/svix.rb

```diff
@@ -32,7 +32,7 @@ def initialize(auth_token, options = SvixOptions.new)
                 regional_url = "https://api.in.svix.com"
             end
 
-            uri = URI(options.server_url || regional_url)
+            uri = URI(regional_url || options.server_url)
```

Comment:
This is on purpose, the option server URL is higher priority and should be. Unless I'm missing something?

Reply:
You are right. If the option server URL has the highest priority then we need to keep everything as it is, and make default value of the server_url nil and instead add the default url in the else condition after line 32

Reply:
Oh, the options have a default value? If so, yeah, indeed a bug. Can you revert back this line and make the other one be null like you suggested?

Reply:
Done